carta/session.py

```diff
@@ -455,7 +455,7 @@ def set_viewer_grid(self, rows, columns, grid_mode=GridMode.FIXED.value):
         columns : {1}
             Number of columns.
         grid_mode : {2}
-            To adopt dynamic grid size mode (``dynamic``) or fixed grid size mode(``fixed``). Default is ``fixed``.
+            To adopt dynamic grid size mode (:obj:`grid_mode=GridMode.DYNAMIC`) or fixed grid size mode(:obj:`grid_mode=GridMode.FIXED`). Default is :obj:`grid_mode=GridMode.FIXED`.
```

Comment:
I think you pasted the wrong thing here -- should be the path.

Reply:
I also think listing the options here isn't necessary, and this should be rephrased -- but it's fine to keep the default. So:
```
The grid mode to adopt. The default is :obj:`carta.constants.GridMode.FIXED`.
```

Reply:
Done in dec7725.